frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SharingPaneButton/SharingPaneButton.module.css

```diff
@@ -1,6 +1,6 @@
 .Container:not([data-css-specificity-hack="🤷"]) {
   cursor: pointer;
-  border-color: var(--mb-color-brand-light);
+  overflow: hidden;
```

Comment:
this file contains CSS changes for this disabled banner
![Screenshot 2024-11-28 at 9 16 44 PM](https://github.com/user-attachments/assets/21bd0bd3-1d3b-4ea3-b44a-16491fe968c2)